RevenueCatUI/Helpers/Variables.swift

```diff
@@ -19,45 +20,50 @@ protocol VariableDataProvider {
 
 }
 
+struct VariableMatch {
+    let variable: String
```

Comment:
Nit: line breaks around per our style guide.
I'd also move this to the bottom of the file since it's an implementation detail and less important than the main definition.

Reply:
you're right! updated